infra/src/backoffice_webapp.tf

```diff
@@ -13,6 +13,8 @@ locals {
     # NextAuthJS
     NEXTAUTH_URL    = "https://selfcare.io.pagopa.it/" # FIXME: move into terraform.tfvars or use a terraform resource/data ref
     NEXTAUTH_SECRET = azurerm_key_vault_secret.bo_auth_session_secret.value
+
+    SELFCARE_API_KEY = data.azurerm_key_vault_secret.selfcare_api_key.value
```

Comment:
**chore:** add `var.backoffice_app.selfcare_base_path` tfvar and add its value as `SELFCARE_BASE_PATH` environment variable

Reply:
15eeeac7a7d56100cf951f344d1ca92aaad3bd6e